app/src/main/AndroidManifest.xml

```diff
@@ -108,9 +108,9 @@
             android:name=".ui.MainActivity"
             android:launchMode="singleTask" />
 
-        <activity
-            android:name=".ui.NewTransferActivity"
-            android:launchMode="singleTask" />
```

Comment:
Commenting only to block this PR while I'm checking if this doesn't break notifications.

Reply:
As seen with @LunarX, it breaks notifications.
But it should be easy to fix, we just need to decide what behavior we prefer.

Reply:
Finally, it was easy to fix, but not easy to find the correct fix.
A lot of circonvolution were involved.